sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -426,17 +427,6 @@ export interface TableInsertEntityHeaders {
     version?: string;
 }
 
-// @public
-export interface TableInsertEntityHeaders {
```

Comment:
This is not a deletion. The TableInsertEntityHeaders was defined twice earlier. Now, one of it is removed.